pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -111,6 +118,13 @@ func (cm *containerManagerImpl) enforceNodeAllocatableCgroups() error {
 		cm.recorder.Eventf(nodeRef, v1.EventTypeNormal, events.SuccessfulNodeAllocatableEnforcement, "Updated limits on system reserved cgroup %v", nc.SystemReservedCgroupName)
 	}
 	if nc.EnforceNodeAllocatable.Has(kubetypes.KubeReservedEnforcementKey) {
+		if len(nc.KubeReservedCgroupName) == 0 {
```

Comment:
Maybe extract the body of if block to a method so that it can be used for line 105 as well.

Reply:
Done